modules/core/shared/src/main/scala/lucuma/core/enums/GmosNorthGrating.scala

```diff
@@ -6,12 +6,14 @@ package core
 package enums
 
 import cats.syntax.eq._
-import coulomb._
+import coulomb.*
+import coulomb.policy.spire.standard.given
+import coulomb.syntax.*
 import eu.timepit.refined.types.numeric.PosInt
 import lucuma.core.math.Angle
 import lucuma.core.math.Coverage
 import lucuma.core.math.Wavelength
-import lucuma.core.math.units._
+import lucuma.core.math.units.{_, given}
```

Comment:
`{*, given}`

Reply:
Hmm, seems our organize imports rule enforces `_` instead of `*` ...

Reply:
- https://github.com/liancheng/scalafix-organize-imports/issues/221

Reply:
Uh haven't seen than